src/modules/project/pages1/projectView/views/body/sections/header/shareModal/views/ProjectShareView.tsx

```diff
@@ -1,97 +1,186 @@
+        <>
+          {t('So far, ')}
+          <Body as="span" color="neutral1.12">
+            {ambassadorsCount}
+          </Body>{' '}
+          <Body as="span" regular>
+            {t('ambassador' + (ambassadorsCount === 1 ? ' has' : 's have') + ' enabled')}
+          </Body>{' '}
+          <Body as="span" color="neutral1.12">
+            {satAmount.toLocaleString()}
+          </Body>{' '}
+          {t('sats in contributions to this project.')}
+        </>
+      )
+    }
 
-  const isImage = validateImageUrl(project.images[0])
+    return ''
+  }
 
   return (
-    <VStack
-      w="100%"
-      border="1px solid"
-      borderColor="neutral1.6"
-      background="neutral1.2"
-      borderRadius={8}
-      overflow={'hidden'}
-      position="relative"
-    >
-      <ProjectShareBanner
-        ref={ref}
-        bannerImage={isImage && project.images[0] ? project.images[0] : project.thumbnailImage || GeyserShareImageUrl}
-        qrCodeValue={projectUrl}
-        qrCodeText={t('View project')}
-        centerLogo={LogoDark}
-        banneText={project.title}
-      />
-
-      <HStack padding={3} width="100%">
-        {generating ? (
-          <Button variant="solid" colorScheme="neutral1" w="full" leftIcon={<Spinner size="sm" />}>
-            {t('Generating banner...')}
+    <VStack w="100%">
+      {/* <VStack border="1px solid" borderColor="neutral1.6" borderRadius="md" width="100%" spacing={2}> */}
+      <VStack
+        bgGradient={isLoggedIn ? 'linear(to-r, primary1.4, indigo.3)' : 'linear(to-r, neutral1.6, neutral1.3)'}
+        p={4}
+        borderRadius="md"
+        width="100%"
+        spacing={2}
+      >
+        <Body size="md" textAlign="center">
+          {!ambassadorsCount ? (
+            <Body>
+              {t('Become the first project')}{' '}
+              <Tooltip label={t('Share your hero link to become a project ambassador')} placement="top">
+                <span style={{ position: 'relative', display: 'inline-block' }}>
+                  <Body as="span" color="neutral1.12" textDecoration="underline dotted" display="inline" bold>
+                    {t('Ambassador')}
+                  </Body>
+                </span>
+              </Tooltip>{' '}
+              {t('by spreading the word and enabling more contributions to this project.')}
+            </Body>
+          ) : (
+            <Body color="neutral1.12" size="md" regular textAlign="center">
+              {t('Become an')}{' '}
+              <Tooltip label={t('Share your hero link to become a project ambassador')} placement="top">
+                <span style={{ position: 'relative', display: 'inline-block' }}>
+                  <Body as="span" color="neutral1.12" textDecoration="underline dotted" display="inline" bold>
+                    {t('Ambassador')}
+                  </Body>
+                </span>
+              </Tooltip>{' '}
+              {t('for this project by spreading the word using your')}{' '}
```

Comment:
For texts that have variables or extra HTML nodes in between we use the <Trans /> component that enables complex translations. Benefit is that the text would be a single one with variables in between, that enables better context for the translators.